### PR TITLE
add Bella soldered keyboard

### PR DESCRIPTION
**Pull request by moyi4681**
add bella soldered keyboard

### CODE REVIEW

File: keyboards/kbdfans/bella/soldered/rules.mk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (drashna) [inline keyboards/kbdfans/bella/soldered/rules.mk:41]: This isn't needed. But bootloader type is. 

```suggestion
# Bootloader  type
BOOTLOADER = atmel-dfu
```

File: keyboards/kbdfans/bella/soldered/rules.mk [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+BOOTMAGIC_ENABLE ?= lite # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= no    # Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes	# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= yes  # Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no
+RGBLIGHT_ENABLE ?= no
```
Comment (drashna) [inline keyboards/kbdfans/bella/soldered/rules.mk:56]:
```suggestion
BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration
MOUSEKEY_ENABLE  = yes  # Mouse keys
EXTRAKEY_ENABLE  = yes  # Audio control and System control
CONSOLE_ENABLE   = no   # Console for debug
COMMAND_ENABLE   = no   # Commands for debug and configuration
SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
NKRO_ENABLE      = yes  # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
AUDIO_ENABLE     = no
RGBLIGHT_ENABLE  = no
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/bella/soldered/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration
+MOUSEKEY_ENABLE  = yes  # Mouse keys
+EXTRAKEY_ENABLE  = yes  # Audio control and System control
+CONSOLE_ENABLE   = no   # Console for debug
+COMMAND_ENABLE   = no   # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE      = yes  # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
+AUDIO_ENABLE     = no
+RGBLIGHT_ENABLE  = no
```
Comment (fauxpark) [inline keyboards/kbdfans/bella/soldered/rules.mk:20]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/bella/soldered/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "bella_soldered", 
+    "url": "", 
+    "maintainer": "kbdfans", 
+    "width": 16.25, 
+    "height": 6.5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/kbdfans/bella/soldered/info.json:8]:
```suggestion
        "LAYOUT_all": {
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**